.changeset/poor-dodos-compare.md

```diff
@@ -0,0 +1,5 @@
+---
+'@aws-amplify/cli-core': patch
+---
+
+prevent CTRL+C handling for pnpm
```

Comment:
at this pace we'll eliminate this prompt altogether.

Npm/npx is also broken in latest versions.

Reply:
Oh you're talking about how the prompt doesn't appear at all. We can consider this since so far it is only working on yarn 4.x from what I've seen.

Reply:
Yeah. It's likely work that should happen separately from this PR, so don't block on it.